config/storage.yml

```diff
@@ -7,7 +7,7 @@ google_development:
   service: GCS
   project: "weg-li"
   bucket: "weg-li-development"
-  credentials: "<%= Rails.root.join("config/secrets/gcloud.json") %>"
+  credentials: <%= Rails.root.join("config/secrets/gcloud.json") %>
```

Comment:
This is the only dubious change. Prettier did not like the quotation marks. Do you think this is a problem @phoet ? I would say it was kinda problematic before, since there are also quotation marks in the string.

Reply:
please ignore files in `config` as they are mostly auto-generated

Reply:
Would it not be faster to just check out the branch and add the changes you want yourself? I mean I am happy to apply the changes from your comments here, but I think it is less productive for both of us.

If you still prefer me to do it: Can you provide me a list of all the files that are auto-generated? I would then put this list into the appropriate `.toolignore` files.